project/Library.scala

```diff
@@ -130,9 +134,17 @@ object Library {
     "ch.epfl.scala" %% "collection-strawman" % "0.9.0"
   )
 
-  val arrow = Seq(
-    ("org.apache.arrow" % "arrow-vector" % Version.arrow)
-      .excludeAll(ExclusionRule("io.netty", "netty-common"))
-      .excludeAll(ExclusionRule("io.netty", "netty-buffer"))
+  val arrowDeps = Seq(
+    "org.apache.arrow" % "arrow-memory" % Version.arrow,
+    "org.apache.arrow" % "arrow-vector" % Version.arrow
   )
+
+  val parquetDeps = Seq(
+    "com.github.mjakubowski84" %% "parquet4s-core" % Version.parquet,
+    "org.apache.hadoop" % "hadoop-client" % Version.hadoopClient,
+    "org.apache.hadoop" % "hadoop-common" % Version.hadoopClient,
+    "org.apache.parquet" % "parquet-hadoop" % Version.parquetCodecs,
```

Comment:
adding hadoop to dependencies opens doors to the hell) Don't do that if you are not 100% sure. Even then)